Makefile.am

```diff
@@ -6702,13 +6702,13 @@ tests += \
 install-tests: $(tests) $(TEST_DATA_FILES)
 	for f in $(tests); do \
 	    if [ -x .libs/$$f ]; then \
-		install -D -m 755 .libs/$$f $(DESTDIR)/$(testsdir)/$$f; \
+		install -D -m 755 $(top_builddir)/.libs/$$f $(DESTDIR)/$(testsdir)/$$f; \
 	    else \
-		install -D -m 755 $$f $(DESTDIR)/$(testsdir)/$$f; \
+		install -D -m 755 $(top_builddir)/$$f $(DESTDIR)/$(testsdir)/$$f; \
```

Comment:
I was not really necessy for the executables. It just seemed more symmetric to add it there as well.

Speaking of the executable, I wonder if we should use libtool here. E.g. taken from install-rootlibexecLTLIBRARIES:
```
$(LIBTOOL) $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=install $(INSTALL) $(INSTALL_STRIP_FLAG) $$list2 "$(DESTDIR)$(rootlibexecdir)";
```

The .libs directory is an implementation detail of libtool.

The full install rule 
```
install-rootlibexecLTLIBRARIES: $(rootlibexec_LTLIBRARIES)
	@$(NORMAL_INSTALL)
	@list='$(rootlibexec_LTLIBRARIES)'; test -n "$(rootlibexecdir)" || list=; \
	list2=; for p in $$list; do \
	  if test -f $$p; then \
	    list2="$$list2 $$p"; \
	  else :; fi; \
	done; \
	test -z "$$list2" || { \
	  echo " $(MKDIR_P) '$(DESTDIR)$(rootlibexecdir)'"; \
	  $(MKDIR_P) "$(DESTDIR)$(rootlibexecdir)" || exit 1; \
	  echo " $(LIBTOOL) $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=install $(INSTALL) $(INSTALL_STRIP_FLAG) $$list2 '$(DESTDIR)$(rootlibexecdir)'"; \
	  $(LIBTOOL) $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=install $(INSTALL) $(INSTALL_STRIP_FLAG) $$list2 "$(DESTDIR)$(rootlibexecdir)"; \
	}
```